appcontainer/BannerSourceModel.ts

```diff
@@ -22,17 +22,25 @@ export class BannerSourceModel extends HoistModel {
     @observable.ref
     bannerModels: BannerModel[] = [];
 
-    MAX_BANNERS = 4;
```

Comment:
Removed cap on banners because would have been complicated to implement and didn't seem necessary.